app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -14,4 +14,18 @@ def provider_collection(manager)
   def provider_object(connection = nil)
     (connection || connection_source.connect).api.projects.find(manager_ref)
   end
+
+  REFRESH_ON_TOWER_SLEEP = 1.second
+  def refresh_in_provider
+    with_provider_object do |project|
+      return unless project.can_update?
+
+      project_update = project.update
+
+      while project_update.finished.blank?
```

Comment:
so if I understand correctly, we need to first do a 'refresh' inside ansible?

I don't really like the sleep here @agrare what do you think? But if we really need to do this, we should have a log in the beginning/end, so we can measure it and it should be in a Timeout block, with possibly configurable timeout time span.

I think AnsibleTower might be the first provider that also has a refresh, so we should probably introduce some pre-refresh statemachine or even a generic task, that would keep the provider refreshed?